.vscode/settings.json

```diff
@@ -22,5 +22,11 @@ Happy linting! 💖
   "playwright.env": {
     "IS_DEV": true
   },
-  "playwright.reuseBrowser": true
+  "playwright.reuseBrowser": true,
+  "tailwindCSS.classAttributes": [
```

Comment:
This allows the Tailwind Plugins intellisense pick up in the following string values